_data/projects/video-hub-app.yml

```diff
@@ -0,0 +1,12 @@
+name: Video Hub App
+desc: The fastest way to browse and search for videos on your computer
+site: https://github.com/whyboris/Video-Hub-App
+tags:
+- oss
+- video
+- angular
+- electron
+- javascript
+upforgrabs:
+  name: Video Hub App
```

Comment:
@whyboris This should be "good first issue" as per the label link that you have mentioned below.

Reply:
Sorry misunderstood 😅 will fix